labs-ios-starter/Model/Apollo/Topic/ContextQuestion.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Context.swift
+//  labs-ios-starter
+//
+//  Created by Kenny on 9/8/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+import Foundation
+
+//used to determine a Question's context
+struct ContextQuestion: Codable {
+    var id: UUID
```

Comment:
Good catch. There's a comment on one of the structs about them all needing to match the backend, and this might change... but that's not clear in every file as it probably should be